recipes/gitdb2/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.0.2" %}
+{% set sha256 = "f2e36d7561e91f30a6a44858756dc020d8f1e81ca6e4185979d5c6c24c648070" %}
+
+package:
+  name: gitdb
+  version: {{ version }}
+
+source:
+  fn: gitdb-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/gitdb2/gitdb2-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - smmap2
+    - setuptools
+
+test:
+  imports:
+    - gitdb
+    - gitdb.db
+
+about:
+  home: https://github.com/gitpython-developers/gitdb
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: 'A pure-Python git object database'
+
+extra:
+  recipe-maintainers:
+    - ngoldbaum
+    - pelson
```

Comment:
It doesn't really make much difference when you have admin rights on conda-forge, right... 😉 

I can't claim to be able to be super responsive on maintenance front 😜 

Basically, if it is there, that's fine. If not, happy to keep it that way.